modules/admin_manual/pages/_partials/maintenance/major_release_note.adoc

```diff
@@ -11,9 +11,13 @@ Here are some examples:
 [cols=">10%,^25%,65%",options="header",stripes=even]
 |===
 |Version
-|Can Upgrade to 10.3.2?
+|Can Upgrade to 10.4.1?
 |Requirements
 
+|10.3.1
```

Comment:
```suggestion
|10.3.2
```

Reply:
The structure is :

![image](https://user-images.githubusercontent.com/24212810/85525841-bf613080-b609-11ea-8e75-a6b2eeac6ef5.png)

I don't know why, I did not make it, but it's always x.0.1.

If you want to change this structure - PR welcome :)

This PR updates the current latest version, which is 10.4.1 right now.

When 10.5.1 is out - this will be the latest version.

Reply:
There is `10.0.10` below, so someone in the past has used the latest patch version of each minor version as examples.
But I will make another PR to adjust that.